testCommon/src/testFixtures/java/com/virtuslab/gitmachete/testcommon/TestProcessUtils.java

```diff
@@ -25,14 +25,15 @@ public static String runProcessAndReturnStdout(Path workingDirectory, int timeou
     String stderr = IOUtils.toString(process.getErrorStream(), StandardCharsets.UTF_8);
     String commandRepr = Arrays.toString(command);
 
-    Assert.assertTrue("Stdout of " + commandRepr + " : " + System.lineSeparator() + " " + stdout,
-        !completed || process.exitValue() != 0);
-    Assert.assertTrue("Stderr of " + commandRepr + " : " + System.lineSeparator() + " " + stderr,
-        !completed || process.exitValue() != 0);
-
-    Assert.assertTrue("command ${commandRepr} has not completed within ${timeoutSeconds} seconds;", completed);
+    Assert.assertTrue(
+        "Stdout of ${commandRepr}: ${System.lineSeparator()} ${stdout};" +
+            "\n Stderr of ${commandRepr}: ${System.lineSeparator() ${stderr};" +
+            "\n command ${commandRepr} has not completed within ${timeoutSeconds} seconds;",
+        completed);
     int exitValue = process.exitValue();
-    Assert.assertEquals("command ${commandRepr} has completed with exit code ${exitValue};", 0, exitValue);
+    Assert.assertEquals("Stdout of ${commandRepr}: ${System.lineSeparator()} ${stdout};" +
```

Comment:
You can extract this shared part to a shared `val`.

Also, does string interpolation work for you finally? if not, pls use `+`s consistently everywhere

Reply:
yes string interpolation work for me finally :) . yes your right let me put Stdout of ${commandRepr}: ${System.lineSeparator()} into val .